specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2019-10-01/analyticsItems_API.json

```diff
@@ -0,0 +1,380 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApplicationInsightsManagementClient",
+    "description": "Azure Application Insights client for saved items.",
+    "version": "2019-10-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/components/{resourceName}/{scopePath}": {
+      "get": {
+        "description": "Gets a list of Analytics Items defined within an Application Insights component.",
+        "operationId": "AnalyticsItems_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ItemScopePathParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ItemScopeParameter"
+          },
+          {
+            "$ref": "#/parameters/ItemTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/IncludeContentParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A list containing 0 or more items associated with the Application Insights component.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ApplicationInsightsComponentAnalyticsItem"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AnalyticsItemList": {
+            "$ref": "./examples/AnalyticsItemList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/components/{resourceName}/{scopePath}/item": {
```

Comment:
This URI is problematic. The segments after the provider name should comprise tuples of resourceTypeName/resourceIdentifier. In this URI:
- /components is a resource type and {resourceName} is the name of the component resource
- {scopePath} does not look like a resource type, but it's segment suggests it is
- /items is in the position of a resourceIdentifier, but looks like a type

Feel free to schedule a meeting with armapireview to discuss.